cli/src/main.rs

```diff
@@ -85,12 +84,12 @@ fn print_help() {
     println!("pass `--version` or `-V` to print version information");
     println!();
     println!("Iroha 2 is configured via environment variables:");
-    println!("    IROHA2_CONFIG_PATH is the location of your `config.json`");
-    println!("    IROHA2_GENESIS_PATH is the location of `genesis.json`");
+    println!("    IROHA2_CONFIG_PATH is the location of your `config.json` or `config.json5`");
+    println!("    IROHA2_GENESIS_PATH is the location of `genesis.json` or `genesis.json5`");
```

Comment:
We don't actually check the extension, and given that the `json5` file is also `json`, it's kind of moot.

Reply:
Yeah, *json* is a valid *json5*, so I think it's okay to use `json5` crate to parse just *json*-files.

Or do you suggest to prohibit non *json*/*json5* extensions?

Reply:
@ilchu suggested it [here](https://github.com/hyperledger/iroha/pull/3030#discussion_r1054465803) so I'll resolve this